jerry-core/debugger/jerry-debugger-ws.c

```diff
@@ -13,8 +13,6 @@
  * limitations under the License.
  */
 
-#ifdef JERRY_DEBUGGER
```

Comment:
The problam is that not everybody use the cmake build system. Some people just use the files (all-in-one build). E.g. esp8266.